client/whisper_engine.go

```diff
@@ -13,13 +15,13 @@ const (
 	// This determines how much audio we will be passing to whisper inference.
 	// We will buffer up to (whisperSampleWindowMs - pcmSampleRateMs) of old audio and then add
 	// audioSampleRateMs of new audio onto the end of the buffer for inference
-	whisperSampleWindowMs = 5000 // 5 second sample window
+	whisperSampleWindowMs = 24000 // 5 second sample window
 	whisperWindowSize     = whisperSampleWindowMs * whisperSampleRateMs
 	// This is the minimum ammount of audio we want to buffer before running inference
 	whisperWindowMinSize = whisperWindowSize / 2
 	// This determines how often we will try to run inference.
 	// We will buffer (pcmSampleRateMs * whisperSampleRate / 1000) samples and then run inference
-	pcmSampleRateMs = 500
+	pcmSampleRateMs = 6000
```

Comment:
Could you also add some context for why you increased this size as well?

Reply:
```suggestion
	pcmSampleRateMs = 500
```

I'd like this value to stay at 500ms however as I want the sample frequency to remain high since this is mean to be used in a real-time context.

Reply:
My pc is not big enough for that big of a window and that fast of a sampling rate 😂

Reply:
> My pc is not big enough for that big of a window and that fast of a sampling rate 😂

Ah that makes sense haha. Im fortunate to have a beefy m1.